db/postgres/database.js

```diff
@@ -0,0 +1,92 @@
+const pg = require('pg');
+
+let client;
+
+if (process.env.NODE_ENV === 'test') {
```

Comment:
@phucciTr Yeah so actually this one I struggled with for awhile before remembering that Jest automatically sets `process.env.NODE_ENV` to 'test' while running. So since my dev environment and Travis both use Jest to run the tests, I can just check if this variable is set. If so, it means Jest is running and I should use the alternate DB instead of production.